community/front-end/ofe/website/ghpcfe/models.py

```diff
@@ -937,9 +944,9 @@ class ClusterPartition(models.Model):
         default="pd-standard",
     )
     boot_disk_size = models.PositiveIntegerField(
-        validators=[MinValueValidator(49)],
+        validators=[MinValueValidator(120)],
         help_text="Boot disk size (in GB)",
-        default=50,
+        default=120,
```

Comment:
What is the reason for changing the default? What is the reason for raising min validator above 50. I believe 50 gb worked previously. Whatever min value is set here forces all compute nodes to adopt that as a minimum. In a large cluster this could be expensive, especially if the on disk storage is not needed.

Reply:
Yep, that make sense. It was misunderstood to when developer was asked to increase disks size, mostly the ask was to increase disk size on the login nodes. Changed back to 50 gb on compute nodes